sdk-python/tests/test_workflow.py

```diff
@@ -814,6 +814,7 @@ def my_entrypoint(thread: WorkflowThread) -> None:
         self.assertEqual(
             wf.compile(),
             PutWfSpecRequest(
+                allowed_updates="ALL",
```

Comment:
Is not clear if we are covering the case when this function is invoked without the allowed_updates argument

Reply:
I'll delegate this to @sauljabin

Reply:
The way protobuf works is that it's not an `optional` field, so if you don't set it, then it is treated as the protobuf "zero" field. That means the enum value `0`; in this case, allowing `ALL` udpates.